### PR TITLE
Simplified arguments to resize methods in packed canvas

### PR DESCRIPTION
**Pull request by sebastianbaginski**
First step to make coordinate systems work in draw method.
This PR only simplifies arguments passed to resize methods.
@marcusnaslund

### CODE REVIEW

File: source/draw/RasterPacked.ooc [hunk 1]
```diff
@@ -31,18 +31,18 @@ RasterPackedCanvas: abstract class extends RasterCanvas {
...
 			memcpy(this target buffer pointer, sourceBuffer, this target stride * this target height)
 		else if (this interpolationMode == InterpolationMode Fast)
-			This _resizeNearestNeighbour(sourceBuffer, this target buffer pointer as T*, sourceBox, resultBox, source stride, this target stride, this target bytesPerPixel)
+			This _resizeNearestNeighbour(sourceBuffer, this target buffer pointer as T*, source, this target, sourceBox, resultBox)
 		else
-			This _resizeBilinear(sourceBuffer, this target buffer pointer as T*, sourceBox, resultBox, source stride, this target stride, this target bytesPerPixel)
+			This _resizeBilinear(sourceBuffer, this target buffer pointer as T*, source, this target, sourceBox, resultBox)
 	}
-	_resizeNearestNeighbour: static func <T> (sourceBuffer, resultBuffer: T*, sourceBox, resultBox: IntBox2D, sourceStride, resultStride, bytesPerPixel: Int) {
+	_resizeNearestNeighbour: static func <T> (sourceBuffer, resultBuffer: T*, source, target: RasterPacked, sourceBox, resultBox: IntBox2D) {
```
Comment (sebastianbaginski) [inline source/draw/RasterPacked.ooc]: Still need to pass the buffer as separate arguments, as rock needs to deduce the `T` implicitly.

Comment (marcusnaslund) [inline source/draw/RasterPacked.ooc]: Isn't it possible to have a `T: Class` argument where you pass the type of `T`?

Comment (sebastianbaginski) [inline source/draw/RasterPacked.ooc]: `T` is a cover here (ColorBgr, ColorMonochrome etc)

Comment (marcusnaslund) [inline source/draw/RasterPacked.ooc]: Right, OK.

**Pull request merged**